lib/card/query/ref_spec.rb

```diff
@@ -20,7 +20,7 @@ def to_sql *args
       field1, field2 = REFERENCE_FIELDS[ dir ]
       cond = type ? ["ref_type='#{type}'"] : []
 
-      sql =  %[select #{field1} as ref_id from card_references]
+      sql =  %[select distinct #{field1} as ref_id from card_references]
```

Comment:
without this, multiple references to the same card can lead to WQL returning duplicate cards.